docs/MadHatter.md

```diff
@@ -1,7 +1,9 @@
-#MadHatter BIO
+#MadHatter BIO - (Class Demo)
 
 **Name:** Toby Schuster
 
 **Occupation:** Engineer
 
 **Location:** Milwaukee, WI USA
+
+**Recreation Activity:** SCCA Racing
```

Comment:
Sounds like an interesting activity. Could you add more specifics about what SCCA stands for? I'm not familiar with it.